src/openproduct/producten/urls.py

```diff
@@ -14,7 +14,21 @@
 description = f"""
 Een API voor Producten.
 
+De Producten Api heeft 1 endpoint `producten`. Hiermee kunnen producten zelf, de bijbehorende eigenaren en documenten worden aangemaakt.
+
+### Product
 Een product is een instantie van een producttype (zie producttypen API), in een product worden onder andere de gegevens van de eigenaar, de benodigde data voor het product en bijvoorbeeld de status vastgelegd.
+Specifieke product data kan worden opgeslagen in de json velden `dataobject` en `verbruiksobject`. Deze velden worden gevalideerd door de `verbruiksobject_schema` & `dataobject_schema` velden van het producttype. ([zie jsonschema](https://json-schema.org))
+De status van een product kan alleen worden veranderd naar de een van de `toegestane statussen` gedefineerd op het producttype.
+
+### Eigenaar
+Aan een product kunnen één of meerdere eigenaren worden gelinkt. Een eigenaar kan een klant (Open Klant), natuurlijk of niet natuurlijke persoon zijn.
+
+### Document
+Een document is een verwijzing naar een document uit de [documenten API](https://vng-realisatie.github.io/gemma-zaken/standaard/documenten/).
```

Comment:
Unfortunately they are not called `document` in the API :(
```suggestion
Een document is een verwijzing naar een `EnkelvoudigInformatieObject` uit de [documenten API](https://vng-realisatie.github.io/gemma-zaken/standaard/documenten/).
```